tritium-caffeine/src/main/java/com/palantir/tritium/metrics/caffeine/CacheStats.java

```diff
@@ -33,24 +35,28 @@
 import org.checkerframework.checker.index.qual.NonNegative;
 
 public final class CacheStats implements StatsCounter, Supplier<StatsCounter> {
+
+    private final CacheMetrics metrics;
     private final String name;
     private final Meter hitMeter;
     private final Meter missMeter;
     private final Timer loadSuccessTimer;
     private final Timer loadFailureTimer;
-    private final Meter evictionsTotalMeter;
     private final ImmutableMap<RemovalCause, Meter> evictionMeters;
-    private final LongAdder totalLoadNanos = new LongAdder();
+    private final ImmutableMap<RemovalCause, Meter> evictionWeightMeters;
+    private final LongAdder totalLoadTime = new LongAdder();
```

Comment:
nit: Renamed this to match the `CacheStats.of` parameter name in order to improve comprehension.